yang/frr-nexthop.yang

```diff
@@ -200,10 +204,45 @@ module frr-nexthop {
           "Identifies a nexthop within a nexthop group; the entries
            are ordered by id value, and the value has no other meaning.";
       }
+      uses frr-nexthop-attributes;
+    }
+  }
 
-      uses frr-nexthop;
+  typedef frr-nexthop-ref {
+    type leafref {
+      path "/frr-nexthop:frr-nexthop/frr-nexthop:entry/frr-nexthop:id";
     }
+    description
+      "Reference to a nexthop-group";
   }
 
-  // End of frr-nexthop-group
+  container frr-nexthop-group {
+    description
+      "A nexthop-group, represented as a list of nexthop objects.";
+
+    list group-entry {
+      key "group-id";
+      description
+        "A list of nexthop objects.";
+      leaf group-id {
+        type uint32;
+        description
+          "Identifies a nexthop within a nexthop group; the entries
+           are ordered by id value, and the value has no other meaning.";
+      }
+      leaf name {
+        type string;
+        description
+          "The nexthop-group name.";
+      }
+      list group-nexthop {
+        key "group-sub-id";
+        leaf group-sub-id {
+          type frr-nexthop-ref;
+          description
+            "A refrence to a staticd-nexthop entry";
```

Comment:
"refrence" - typo?

"staticd" - also a typo?

Reply:
Will correct it.